doc/admin/installation.md

```diff
@@ -3,6 +3,8 @@
 DAOS runs on both Intel 64 and ARM64 platforms and has been successfully tested
 on CentOS 7, OpenSUSE Leap 15.1, and Ubuntu 20.04 distributions.
 
+The majority of testing has been performed on Centos 7.7 and SLES 15, with Centos being used in the majority of the test cycles.
```

Comment:
Is this documentation specific to 1.0.x?  There was no SLES/Leap testing of that release AFAIK.

Regardless, I'm not sure if or how much it matters but any SLES/Leap testing would have been done on Leap, not SLES.